global.d.ts

```diff
@@ -36,6 +36,14 @@ declare global {
   }
 
   namespace Mocha {
+    interface SuiteFunction {
```

Comment:
Not a request for changes just wanting to understand the rationale for including these 2 definitions here.

Reply:
Our metadata UI also allows specifying filters at the describe level, only in the 3 argument form of `title, metadata, function` So this just prevents type issues when using a metadata object in `describe` or `context`. I just don't think we've used metadata at the suite level much before or we haven't needed it in TS test yet. But the transactions tests will always need to filter up to 4.2 maybe 4.4 so it makes sense to specify it higher up.